server/app.go

```diff
@@ -233,27 +233,26 @@ func startHttpServer(a *App, ctx context.Context) error {
 
 func startAmc(a *App, ctx context.Context) error {
 	listener := eventlistener.NewEventListener(a.config.AmcConfig.Url, a.events)
-	log.Info().Msgf("Connecting to the action monitor on %s", a.config.AmcConfig.Url)
+	// setup reconnection options
+	listener.ReconnectionAttempts = 5
```

Comment:
move to  config and set default like here: https://github.com/DaoCasino/casino-backend/blob/develop/config.go#L14